.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Unit tests
+on:
+  pull_request:
+  push:
+    branches:
+    - qa/**
+    - stable/**
+jobs:
+  unit-tests:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [7.2, 7.3, 7.4, 8.0]
```

Comment:
Let's remove the PHP 8.0 test until we have time to update the code so the tests pass.